cpp/src/io/avro/avro_gpu.cu

```diff
@@ -190,7 +198,44 @@ avro_decode_row(schemadesc_s const* schema,
           skip += schema[i].count;  // Should always be 1
         }
       } break;
+
+      case type_duration: {
+        // A duration logical type annotates Avro fixed type of size 12, which
+        // stores three little-endian unsigned integers that represent durations
+        // at different granularities of time. The first stores a number in
+        // months, the second stores a number in days, and the third stores a
+        // number in milliseconds.
+        CUDF_UNREACHABLE("avro type 'duration' not yet implemented");
+      } break;
+
+      // N.B. These aren't handled yet, see the discussion on
+      //      https://github.com/rapidsai/cudf/pull/12788.  The decoding logic
+      //      is correct, though, so there's no harm in having them here.
+      case type_timestamp_millis: [[fallthrough]];
+      case type_timestamp_micros: [[fallthrough]];
+      case type_local_timestamp_millis: [[fallthrough]];
+      case type_local_timestamp_micros: [[fallthrough]];
+      case type_time_millis: [[fallthrough]];
+      case type_time_micros: {
+        // N.B. time-millis is stored as a 32-bit int, however, cudf expects an
+        //      int64 for DURATION_MILLISECONDS.  From our perspective, the fact
+        //      that time-millis comes from a 32-bit int is hidden from us by
+        //      way of the zig-zag varint encoding, so we can safely treat them
+        //      both as int64_t.  Everything else is 64-bit in both avro and
+        //      cudf.
+        CUDF_UNREACHABLE("avro type yet implemented");
```

Comment:
typo fix
```suggestion
        CUDF_UNREACHABLE("avro type `time-millis` not yet implemented");
```

Reply:
Fixed.  Changed it to `time/timestamp` as it's not just `time-millis`.